storage/queue.go

```diff
@@ -170,9 +179,26 @@ func (bq *baseQueue) Add(repl *Replica, priority float64) error {
 // not be added, as the replica with the lowest priority will be
 // dropped.
 func (bq *baseQueue) MaybeAdd(repl *Replica, now proto.Timestamp) {
+	// Load the system config.
+	cfg, err := bq.gossip.GetSystemConfig()
```

Comment:
this part hasn't changed much since before, we already fetched the system config for all `shouldQueue` and `process` methods. It's just that now we also do it for `range_gc_queue` and `verify_queue`.
Still, I agree, this is way too much work, but @bdarnell's right, we should put this in gossip itself. The system config is special enough that decoding it there and saving the unmarshaled message makes sense.